### PR TITLE
Doco for changes to SP 1.4.0

### PR DESCRIPTION
**Pull request by petersgiles**
to be merged on release day

### CODE REVIEW

**Comment by petersgiles** [comment]
@SimonCropp release has occurred, I've added new images and change text. Wasn't as bad as I thought

Please merge

**Comment by mikeminutillo** [comment]
I'm reviewing now and then I'll merge

**Comment by mikeminutillo** [comment]
These aren't related to your changes (I don't think) but to the page as a whole.

> The first tab shows error groups.

The second tab is not mentioned in the docs at all that I can see.

There is a Related Articles section in the markdown content but the docs engine has that as a built in capability that always get added to the bottom. @Particular/docs-maintainers in this case should we switch to using the built in capability?

**Comment by mikeminutillo** [comment]
Related articles can be referenced using this header https://github.com/particular/docs.particular.net#related

**Comment by mikeminutillo** [comment]
Pulled "Related articles" out into a separate issue (https://github.com/Particular/docs.particular.net/issues/1238) so we can merge this.

File: servicepulse/intro-failed-messages.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: Failed Message Monitoring in ServicePulse
 summary: Describes how ServicePulse detects and monitors failed messages, and how you can resend messages for reprocessing, or archive repeatedly failing messages
 tags:
 - ServicePulse
+reviewed: 2016-03-02
```
Comment (SimonCropp) [inline servicepulse/intro-failed-messages.md]: if u want to mark it as reviewed then remove the personal voice. ie "you" etc

File: servicepulse/intro-failed-messages.md [hunk 2]
```diff
@@ -26,10 +27,12 @@ The first tab shows error groups. A group is a set of failed messages where the
 
 * a **Title** made up of the **Exception Type** and **Call Site** where the failure occurred.
 * a **Count** of how many unresolved messages there are in the group.
-* the **First Fail** time and the **Last Fail** time indicating when the first and most recent unresolved errors occurred respectively.
+* the **First Failure** time and the **Latest Failure** time indicating when the first and most recent unresolved errors occurred respectively.
 * **Actions** which can be used to Archive or Retry an entire group of messages (see below).
```
Comment (SimonCropp) [inline servicepulse/intro-failed-messages.md]: why do some points here start with "the" others with "a" and the last with nothing

File: servicepulse/intro-failed-messages.md [hunk 3]
```diff
@@ -26,10 +27,12 @@ The first tab shows error groups. A group is a set of failed messages where the
 
 * a **Title** made up of the **Exception Type** and **Call Site** where the failure occurred.
 * a **Count** of how many unresolved messages there are in the group.
-* the **First Fail** time and the **Last Fail** time indicating when the first and most recent unresolved errors occurred respectively.
+* the **First Failure** time and the **Latest Failure** time indicating when the first and most recent unresolved errors occurred respectively.
```
Comment (SimonCropp) [inline servicepulse/intro-failed-messages.md]: why group these two?

File: servicepulse/intro-failed-messages.md (hunk 1 above)
Comment (mikeminutillo) [inline servicepulse/intro-failed-messages.md]: I thought I had. I picked up a couple more on a re-review but I missed the summary.

Comment (SimonCropp) [inline servicepulse/intro-failed-messages.md]: also one at the end

File: servicepulse/intro-failed-messages.md [hunk 4]
```diff
@@ -22,14 +23,16 @@ To see a detailed display of the failed messages, click the Failed Messages indi
 ![Failed Message Groups Page](intro-failed-messages-failed-groups-page.png)
 
 
-The first tab shows error groups. A group is a set of failed messages where the same **Exception Type** has been thrown from the method in your solution. Each group has:
+The first tab shows error groups. A group is a set of failed messages where the same **Exception Type** has been thrown from the same method in your solution. Each group has:
```
Comment (SimonCropp) [inline servicepulse/intro-failed-messages.md]: `the solution`

File: servicepulse/intro-failed-messages.md [hunk 5]
```diff
@@ -55,7 +58,7 @@ If a message fails repeated retry attempts, an indication is added, including th
   
 ![Repeated failure indication](images/failed-messages-repeated-failure.png)
 
-NOTE: Tracking the number of repeated retry attempts may be significant to your specific solution implementation, since each retry may invoke custom and/or third party logic that may not participate in the NServiceBus endpoint transactional  processing (and is therefore not rolled back on processing failure).
+NOTE: Tracking the number of repeated retry attempts may be significant, since each retry may invoke custom and/or third party logic that may not participate in the NServiceBus endpoint transactional  processing (and is therefore not rolled back on processing failure).
```
Comment (SimonCropp) [inline servicepulse/intro-failed-messages.md]: can this be split into two sentences?

File: servicepulse/intro-failed-messages.md (hunk 2 above)
Comment (mikeminutillo) [inline servicepulse/intro-failed-messages.md]: Each one is a continuation of "Each group has:" which precedes this list.

File: servicepulse/intro-failed-messages.md (hunk 5 above)
Comment (petersgiles) [inline servicepulse/intro-failed-messages.md]: maybe...to be honest I only have a vague idea of what its trying to say. It kind of implies 'significant' is a metric for  a reason that looks to be unrelated to the users intent.

Comment (SimonCropp) [inline servicepulse/intro-failed-messages.md]: hence i asked. since i find it confusing

Comment (petersgiles) [inline servicepulse/intro-failed-messages.md]: should we just remove it

File: servicepulse/intro-failed-messages.md [hunk 6]
```diff
@@ -55,7 +59,7 @@ If a message fails repeated retry attempts, an indication is added, including th
   
 ![Repeated failure indication](images/failed-messages-repeated-failure.png)
 
-NOTE: Tracking the number of repeated retry attempts may be significant to your specific solution implementation, since each retry may invoke custom and/or third party logic that may not participate in the NServiceBus endpoint transactional  processing (and is therefore not rolled back on processing failure).
+NOTE: The number of retry attempts for a message can be significant if the handler for that message is not [idempotent](http://docs.particular.net/nservicebus/concept-overview#idempotence). Any processing attempt that invokes logic that does not participate in the NServiceBus transactional processing will not be rolled back on processing failure.
```
Comment (mikeminutillo) [inline servicepulse/intro-failed-messages.md]: @SimonCropp How about this?

**Comment by SimonCropp** [comment]
@petersgiles @mikeminutillo thanks, quality work
